views/settings.php

```diff
@@ -24,25 +46,40 @@
 
 	<?php else : ?>
 
-		<form method="post" action="options.php">
-			<?php settings_fields( 'wp101' ); ?>
+		<div id="wp101-settings-api-key-form" <?php echo ( $api_key ) ? 'class="hide-if-js"' : ''; ?>>
```

Comment:
`$api_key` can be returned as `null` if the option has never been set. `( $api_key )` will attempt to cast that to a bool which could result in an error. Best to do a proper `! empty()` check to avoid that possibility.

Reply:
Not overly concerned since a `null` API key would still evaluate as false-y, but it never hurts to be more explicit in type checks. Updated.